components/eamxx/src/physics/mam/eamxx_mam_microphysics_process_interface.cpp

```diff
@@ -720,6 +819,24 @@ void MAMMicrophysics::run_impl(const double dt) {
   auto calday = ts2.frac_of_year_in_days() +
                 1;  // Want day + fraction; calday 1 == Jan 1 0Z
   shr_orb_decl_c2f(calday, eccen, mvelpp, lambm0, obliqr, &delta, &eccf);
+  constexpr int num_modes = mam4::AeroConfig::num_modes();
+  constexpr int gas_pcnst = mam_coupling::gas_pcnst();
+  constexpr int nqtendbb  = mam_coupling::nqtendbb();
+
+  // FIXME: I believe Balwinder add this array somewhere in mam4xx or eamxx.
```

Comment:
Yes, here it is: https://github.com/eagles-project/mam4xx/blob/2eee1988b1e6488c904e0a28564bdcadfa190d34/src/mam4xx/gas_chem_mechanism.hpp#L23